src/components/MainForm/Mainform.tsx

```diff
@@ -40,25 +142,15 @@ interface IFormInput {
 
 export default function Mainform() {
   const [open, setOpen] = React.useState(false);
-  // const [age, setAge] = React.useState<number | string>('');
-
-  // const handleChange = (event: SelectChangeEvent<typeof age>) => {
-  //   setAge(Number(event.target.value) || '');
-  // };
+  const history = useHistory();
+  const [country, setCountry] = React.useState<any>(countries.USA);
 
-  const handleClickOpen = () => {
-    setOpen(true);
+  const FlagsChanger = (country: any) => {
+    setCountry(country);
+    setOpen(false);
   };
 
-  const handleClose = (
-    event: React.SyntheticEvent<unknown>,
-    reason?: string
-  ) => {
-    if (reason !== "backdropClick") {
-      setOpen(false);
-    }
-  };
-  let pakFlag = "🇵🇰";
+  const MuiClasses = useStyles();
 
```

Comment:
Not a good name , name it formClasses or something more clear

Reply:
Done